config/diagnostics/ssh/config.yaml

```diff
@@ -12,37 +12,37 @@ contours: 21
 
 # Models for which sea surface height (SSH) variability is to be evaluated.
 models:
-#   - name: ICON
-#     experiment: ssp370
-#     source: daily-hpz10-oce2d
-#     regrid: r005
-#     variable: zos
-#     zoom: 10
-#     timespan: ['2020-01-20', '2025-05-30']
+  #- name: ICON
+  #  experiment: ssp370
+  #  source: daily-hpz10-oce2d
+  #  regrid: r005
+  #  variable: zos
+  #  zoom: 10
+  #  timespan: ['2020-01-20', '2025-05-30']
 # #    timespan: ['2020-01-01', '2020-12-31']
 
-  - name: IFS-NEMO
+  - name: IFS-FESOM
     experiment: ssp370
-    source: daily-r025-oce2d
-    regrid: r005
+    source: lra-r100-monthly
```

Comment:
please bear in mind that the LRA is not the target of this diagnostic. It can be used for the testing purposes, but this diagnostic is from the frontier set so that it should be able to work with high resolution data.